docs/source/caching.mdx

```diff
@@ -177,3 +177,38 @@ store.withinReadWriteTransaction({ transaction in
   }
 })
 ```
+
+### delete
+
+Delete functionality is limited at this time. There are presently two options available on `NormalizedCache` which are supported by the included implementations of it: 
+
+1. `clear` - Removes everything from the cache immediately. This is basically the "Nuke it from orbit, it's the only way to be sure" option. 
+2. `removeRecord(for:)` Removes a single record for the given `CacheKey`. 
+
+`removeRecord(for:)` will only remove things at the level of a record - that is, it cannot remove individual fields from a record, and it cannot remove outside references **to** a record. 
+
+```swift
+// Assuming we start with records of the following format:
```

Comment:
I think in order to be understandable, this needs a much more detailed explanation of cache normalization and the structure of records. It may make sense to move the implementation details to a separate section and focus on the API and observable behavior here.

Reply:
Gonna work on this bit tomorrow when I have a couple functioning brain cells